Containers/nextcloud/Dockerfile

```diff
@@ -24,10 +24,12 @@ RUN set -ex; \
     addgroup -g 33 -S www-data; \
     adduser -u 33 -D -S -G www-data www-data; \
     \
-# entrypoint.sh and cron.sh dependencies
+# entrypoint.sh and cron dependencies
     apk add --no-cache \
         rsync \
+        cronie \
     ; \
+    echo '*/5 * * * * php -f /var/www/html/cron.php' | tee /etc/crontabs/www-data; \
```

Comment:
I wonder will cronie keep environmental varialbes? Otherwise php will probably run into an issue?

Reply:
the cron.php requires env?

Reply:
Yes due to https://github.com/nextcloud/all-in-one/blob/27dbeec29d720eb19b99c2e47e251e3cb92c2882/Containers/nextcloud/Dockerfile#L105

Reply:
but thats not an env?

Reply:
It is -> `'` is used and not `"`

Reply:
![image](https://github.com/nextcloud/all-in-one/assets/42591237/19a5c04a-c432-4b8c-a9d3-8feb78eaab9b)

Reply:
ahhh now I understand, then lets close this PR, even if I think that cronie would be a better implementation then looping shell script in a while true loop in combination with sleep, if you understand what I mean

Reply:
> even if I think that cronie would be a better implementation

Yes I agree